libraries/standard/http/src/private/http_client_internal.h

```diff
@@ -48,8 +48,11 @@
 #define HTTP_HEADER_END_INDICATOR               "\r\n\r\n"
 #define HTTP_HEADER_END_INDICATOR_LEN           ( sizeof( HTTP_HEADER_END_INDICATOR ) - 1 )
 #define HTTP_HEADER_ADD_FORMAT                  "%.*s" HTTP_HEADER_FIELD_SEPARATOR "%.*s" HTTP_HEADER_LINE_SEPARATOR
-#define CARRIAGE_RETURN_CHARACTER               '\r'
-#define NEWLINE_CHARACTER                       '\n'
+#define HTTP_REQUEST_LINE_FORMAT                "%.*s %.*s %.*s"
```

Comment:
We are not using this anywhere so delete.

Reply:
Are you referring to the HTTP_REQUEST_LINE_FORMAT? If you are then yes I agree that unused definitions are code rot that needs to be removed?